api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -214,15 +222,15 @@ public void localize(String googleProject, String clusterName, Map<String, Strin
   public StorageLink createStorageLink(
       String googleProject, String clusterName, StorageLink storageLink) {
     NotebooksApi notebooksApi = notebooksApiProvider.get();
-    return retryHandler.run(
+    return notebooksRetryHandler.run(
         (context) -> notebooksApi.welderCreateStorageLink(googleProject, clusterName, storageLink));
   }
 
   @Override
   public boolean getNotebooksStatus() {
     try {
-      new StatusApi().getSystemStatus();
-    } catch (ApiException e) {
+      serviceInfoApiProvider.get().getSystemStatus();
```

Comment:
APparently this status API was exclusively checking prod... nice... Fixed that, which should resolve the integration test issues.